src/Plaid/Link/OnSuccessMetadata.cs

```diff
@@ -0,0 +1,156 @@
+			Justification = "Matching the data structure from the Plaid API")]
+		public AccountInfo[] Accounts { get; set; } = null!;
+
+		/// <summary>
+		/// The account object is not null in the onSuccess callback when 
+		/// your users connect a specific bank account via the Select Account 
+		/// pane in Link. You can enable this behavior in the Plaid Dashboard 
+		/// or by integrating with all Auth features.
+		/// 
+		/// Note: If Select Account is enabled for multiple accounts, account 
+		/// will contain only the first account. All accounts selected by the 
+		/// user will be present in the accounts object.
+		/// </summary>
+		[JsonProperty("account")]
+		public AccountInfo Account { get; set; } = null!;
+
+		/// <summary>
+		/// Gets or sets the institution.
+		/// </summary>
+		/// <value>The institution.</value>
+		[JsonProperty("institution")]
+		public InstitutionInfo Institution { get; set; } = null!;
+
+
+		/// <summary>
+		/// Represents an <see cref="Entity.Account"/> metadata.
+		/// </summary>
+		[System.Diagnostics.CodeAnalysis.SuppressMessage(
+			"Design",
+			"CA1034:Nested types should not be visible",
+			Justification = "Required for Json deserialization?")]
+		public class AccountInfo
+		{
+			/// <summary>
+			/// Gets or sets the <see cref="Going.Plaid.Entity.Account"/> identifier.
+			/// </summary>
+			/// <value>The identifier.</value>
+			[JsonProperty("id")]
+			public string Id { get; set; } = null!;
+
+			/// <summary>
+			/// Gets or sets the <see cref="Going.Plaid.Entity.Account"/> name.
+			/// </summary>
+			/// <value>The name.</value>
+			[JsonProperty("name")]
+			public string Name { get; set; } = null!;
+
+			/// <summary>
+			/// The last 2-4 alphanumeric characters of an account's official account number.
+			/// </summary>
+			/// <remarks>The mask may be non-unique between an Item's accounts, it may also not match the mask that the bank displays to the user.</remarks>
+			[JsonProperty("mask")]
+			public string Mask { get; set; } = null!;
+
+			/// <summary>
+			/// Gets or sets the type.
+			/// </summary>
+			/// <value>The type.</value>
+			[JsonProperty("type")]
+			[Newtonsoft.Json.JsonConverter(typeof(Newtonsoft.Json.Converters.StringEnumConverter))]
+			public Going.Plaid.Entity.AccountType? Type { get; set; } = null!;
+
+			/// <summary>
+			/// Gets or sets the type of the sub.
+			/// </summary>
+			/// <value>The type of the sub.</value>
+			[JsonProperty("subtype")]
+			[Newtonsoft.Json.JsonConverter(typeof(Newtonsoft.Json.Converters.StringEnumConverter))]
+			public Going.Plaid.Entity.AccountSubType? SubType { get; set; } = null!;
+		}
+
+
+		/// <summary>
+		/// Represents an <see cref="Entity.Institution"/> metadata.
+		/// </summary>
+		[System.Diagnostics.CodeAnalysis.SuppressMessage(
+			"Design",
+			"CA1034:Nested types should not be visible",
+			Justification = "Required for Json deserialization?")]
+		public class InstitutionInfo
```

Comment:
This appears to be a duplicate of the existing `Institution` class. Any reason for this specifically?

Reply:
It's a subset of the properties plus an Id property. I know... friggin' annoying they didn't just use the same data structure

Reply:
Derp! Ok, update comments in the xdocs, and add a remark about duplicate of the real type for such and such.

Reply:
oops.. I meant o put this comment on the AccountInfo feedback but, same comment anyway =)